modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -306,17 +341,32 @@ private String _formatJava(
 		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
 
 		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
-			if (!childJavaTerm.isJavaMethod()) {
-				continue;
+			String javaContent = null;
+
+			Matcher matcher = null;
+
+			if (childJavaTerm.isJavaMethod()) {
+				JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+				javaContent = javaMethod.getContent();
+
+				Pattern pattern = _getPattern(jsonObject);
+
+				matcher = pattern.matcher(javaContent);
```

Comment:
@TamyrisTorres, these lines can be moved out the `if ... else if ...` statements, please see below:
```
String javaContent = null;

if (childJavaTerm.isJavaMethod()) {
	JavaMethod javaMethod = (JavaMethod)childJavaTerm;

	javaContent = javaMethod.getContent();
}
else if (childJavaTerm.isJavaVariable()) {
	JavaVariable javaVariable = (JavaVariable)childJavaTerm;

	javaContent = javaVariable.getContent();
}

Pattern pattern = _getPattern(jsonObject);

Matcher matcher = pattern.matcher(javaContent);
```
Can you please perform this optimization?